test/e2e/daemonset_test.go

```diff
@@ -171,7 +171,7 @@ func getVertxDeployment(namespace string, selector map[string]string) *appsv1.De
 				},
 				Spec: corev1.PodSpec{
 					Containers: []corev1.Container{{
-						Image: "jaegertracing/vertx-create-span:operator-e2e-tests",
+						Image: "quay.io/maistra/jaeger-vertx-create-span:0.0-ibm-z",
```

Comment:
@madeelrh we should pass the image details as an environment variable and the default image should be `jaegertracing/vertx-create-span:operator-e2e-tests`
like this https://github.com/jaegertracing/jaeger-operator/blob/master/test/e2e/utils.go#L43~L52